parking/src/Ticket.py

```diff
@@ -0,0 +1,86 @@
+import time
+from parking.models import TariffPlan, carDataDetails
+	
+class Ticket:
+    ''' Class used for update/create tariff plans and for
+	    generating tickets with fare details for cars at exit.'''
+		
+    def __init__(self, location) :
+        '''Constructor method'''
+        self.outTime = time.time()
+        car = self.__findVehical(location)
```

Comment:
don't load data inside object constructors, please inject any complex structure so this way if creation of mentioned complex data/structure raises an error it doesn't belong to class instantiation